### PR TITLE
add an example for using the net/http adapter

### PR DESCRIPTION
**Pull request by joeshaw**

### CODE REVIEW

File: _examples/http-adapter/fastly.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+authors = ["oss@fastly.com"]
+description = ""
+language = "go"
+manifest_version = 2
+name = "example"
+
+[local_server]
+  [local_server.backends]
+    [local_server.backends.ipv4]
+      url = "https://ipv4.joeshaw.org"
```
Comment (dgryski) [inline _examples/http-adapter/fastly.toml:13]: We use `httpbin.org` for the other examples.  It has an `/ip` end-point we can use.

**Review by dgryski — CHANGES_REQUESTED** [review]
+1 for more examples.  Is there a way to turn this into a unit test (or add a unit test)?

File: _examples/http-adapter/fastly.toml (hunk 1 above)
Comment (joeshaw) [reply _examples/http-adapter/fastly.toml]: 👍 fixed up in 7af48a8

**Comment by joeshaw** [comment]
> Is there a way to turn this into a unit test (or add a unit test)?

I do have a unit test for the adapter in https://github.com/fastly/compute-sdk-go/blob/main/fsthttp/adapter_test.go.  I could create an integration test which uses this, but as it's entirely an SDK thing (no runtime interaction) I'm not sure it buys us anything.

**Pull request merged**